scripts/compile_bundle.sh

```diff
@@ -26,35 +27,42 @@ if [ ! -f ${DPCPP_ENV} ]; then
     echo "DPC++ compiler environment ${DPCPP_ENV} doesn't seem to exist."
     exit 2
 fi
+source ${DPCPP_ENV}
+
 ONEMKL_ENV=${ONEMKL_ROOT}/env/vars.sh
 if [ ! -f ${ONEMKL_ENV} ]; then
     echo "oneMKL environment ${ONEMKL_ENV} doesn't seem to exist."
     exit 3
 fi
+source ${ONEMKL_ENV}
 
 # Check existance of required Linux commands
-which python > /dev/null 2>&1
-if [[ $? -ne 0 ]]; then
-    echo "Error: linux command \"python\" not found."
-    exit 4
-fi
-which git > /dev/null 2>&1
-if [[ $? -ne 0 ]]; then
-    echo "Error: linux command \"git\" not found."
-    exit 5
-fi
-which patch > /dev/null 2>&1
-if [[ $? -ne 0 ]]; then
-    echo "Error: linux command \"patch\" not found."
-    exit 6
-fi
-which pkg-config > /dev/null 2>&1
-if [[ $? -ne 0 ]]; then
-    echo "Error: linux command \"pkg-config\" not found."
+for APP in python git patch pkg-config nproc bzip2; do
+    command -v $APP || (echo "Error: Command \"${APP}\" not found." ; exit 4)
+done
+
+# Check existance of required libs
+for LIB_NAME in libpng libjpeg; do
+    pkg-config --exists $LIB_NAME || (echo "Error: \"${LIB_NAME}\" not found in pkg-config." ; exit 5)
+done
+
+# set CC if not already defined
+if [ -z "${CC-}" ]; then
+    export CC="$(command -v gcc)" || (echo "Error: gcc not found" ; exit 6)
+fi
+
+if [ $($CC -dumpversion) -ne 11 ]; then
+    echo "Error: GCC version 11 required"
```

Comment:
Would you dump gcc version out to notify users the active gcc version rather than check and exit if it is not gcc 11? It is highly recommended to check installation guide first for requirements, including driver version, oneAPI basekit version, etc.

Reply:
i was adopting the changes from https://github.com/intel/intel-extension-for-pytorch/pull/331 as they reported issues with other gcc versions?

https://intel.github.io/intel-extension-for-pytorch/cpu/latest/tutorials/installation.html#system-requirements

OK, installation guide: "**Recommend** using GCC 11", so i would adopt this into a "warning" instead.

Reply:
Since checking gcc with a specific version in this script brings extra maintenance costs. I'm wondering if just printing the gcc version out could notify users to check the installation guide or not.